Project.toml

```diff
@@ -4,7 +4,7 @@ uuid = "4d7a3746-15be-11ea-1130-334b0c4f5fa0"
 version = "0.13.0"
 
 [compat]
-AMDGPU = "0.5"
+AMDGPU = "0.5, 0.6, 0.7"
```

Comment:
Could you please also add the newly released 0.8 and make sure that all works as it should?